sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/CHANGELOG.md

```diff
@@ -4,10 +4,16 @@
 
 ### Features Added
 
+* Added NET6 target framework.
```

Comment:
.NET 6 is already working before this PR. Does it qualify as a "feature added" entry?

Reply:
I think the "feature added" is appropriate.
Yes a .NET6 app could reference this library, but it's new that the library directly targets this framework.

This is same message we used when we added .NET6 to the Exporter a few months ago.
https://github.com/Azure/azure-sdk-for-net/blob/2e5c4d53344e3f833beea1ae86c245a000de4ed1/sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md?plain=1#L44-L49

Reply:
Shall we include the reason "to support Trimming" in this new entry as well since it's the whole point of having to add the target framework: to be in sync with Exporter to support Trimming?

Reply:
No, we don't support Trimming in the Distro. That's not why we're making that change here.